cwd/editor.js

```diff
@@ -1,3 +1,60 @@
+const update = (btn) => {
+  const viewNum = btn.id.substring(8, 9);
+  const gaugeNum = btn.id.substring(10, 11);
+  const msgNum = btn.id.substring(12, );
+
+  const prob = [];
+  for (const x of Array(5).keys()) {
+    prob.push($('#prob-' + (btn.id.substring(8, )) + "-" + (x + 1)).val());
+  }
+
+  const settings = {
+    "url": views[viewNum - 1] + "gauges/" + gaugeNum + "/messages/" + msgNum,
+    "type": "POST",
+    "timeout": 0,
+    "headers": {
+      "Content-Type": "application/x-www-form-urlencoded"
+    },
+    "data": {
+      "message": "\"" + $('#input-' + (btn.id.substring(8, ))).val() + "\"",
+      "probability": `\[${prob}\]`,
+      "pass": "\"" + $('#passIn').val() + "\""
```

Comment:
I feel like using `toString` might make more intuitive sense for what you are trying to achieve here.

Reply:
This is still a relevant point, but it doesn't matter.